test/libsolidity/smtCheckerTests/imports/import_base.sol

```diff
@@ -13,16 +13,10 @@ pragma experimental SMTChecker;
 import "base";
 contract Der is Base {
 	function g(uint y) public {
+		require(x < 10);
```

Comment:
Added restriction to work around Spacer's nondeterminism

Reply:
Sounds like a fix for a different issue? Could you please add a comment in the test case describing motivation?

Reply:
It's a fix for nondeterministic behaviour of Spacer on different platforms on this particular example.
We encounter the problem quite often if there is a change in the encoding (as is the case of this PR).

Basically, with the change in the encoding of this PR, Spacer exhibits different behaviour on different platforms on this example, so it would be impossible to specify a single expected outcome.
So I added the constraint to restrict the search space in the hope of suppressing the non-determinism. 
I can add a comment with the explanation.

Reply:
Sounds good! It would be nice to document this somehow as a comment imo.

Reply:
I've added the comment in the test case.